test/Domain/LeanCode.DomainModels.EF.Tests/ModelBuilderExtensionsTests.OptimisticConcurrency.cs

```diff
@@ -27,18 +26,22 @@ public void When_IOC_is_implemented_implicitly_DateModified_has_correctly_config
     {
         using (var db = new ImplicitlyContext())
         {
-            var ent = db.Model.FindEntityType(typeof(ImplicitlyImplemented).FullName);
-            var prop = ent.FindProperty("DateModified");
-            Assert.Equal("<DateModified>k__BackingField", prop.FieldInfo.Name);
+            var ent = db.Model.FindEntityType(typeof(ImplicitlyImplemented).FullName!);
+            var prop = ent!.FindProperty("DateModified");
+            Assert.Equal("<DateModified>k__BackingField", prop!.FieldInfo!.Name);
         }
     }
 
     [Fact]
     public void When_DateModified_is_implemented_differently_Fails()
     {
-        using (var db = new WrongDateModifiedContext())
+        WrongDateModifiedContext db;
+
+        using (db = new WrongDateModifiedContext())
```

Comment:
What's the reason for this change?

Reply:
https://www.jetbrains.com/help/rider/AccessToDisposedClosure.html although it might not address the issue it silences rider warning. Maybe explicit supress would be better?

Reply:
Yes, this is misleading on Resharper part. There was nothing wrong in the original code.